src/NServiceBus.Gateway.PersistenceTests/NServiceBus.Gateway.PersistenceTests.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net452;netcoreapp2.1</TargetFrameworks>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>$(SolutionDir)NServiceBus.snk</AssemblyOriginatorKeyFile>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <PackageId>NServiceBus.Gateway.PersistenceTests.Sources</PackageId>
+    <Description>Tests for gateway deduplication storage seam implementations</Description>
+    <IsPackable>true</IsPackable>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" PrivateAssets="All" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.12.0" PrivateAssets="All" />
```

Comment:
If we ship this as a package would it make sense to make this a range like

```
<PackageReference Include="NUnit" Version="[3.12.0, 4.0.0)" />
```

like see https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.TransportTests/NServiceBus.TransportTests.csproj#L18

Reply:
this has PrivateAssets=All so it won't be listed as a dependency

Reply:
Shouldn't this enforce NUnit given that it is required and therefore not have private assets?

Reply:
I think this is how core does it, I guess the adapter is required to make the tests actually run but not for compilation. I'm not familiar with the test adapter package release strategy tbh so not sure it makes sense to change this atm.

Reply:
I'm talking about NUnit.